app/controllers/sign_up/completions_controller.rb

```diff
@@ -34,5 +34,9 @@ def loa3?
     def service_provider_attributes
       { loa3: sp_session[:loa3], service_provider_name: decorated_session.sp_name }
     end
+
+    def event_name
+      loa3? ? 'IDV_COMPLETION' : 'USER_REGISTRATION'
+    end
```

Comment:
What do you think about making these analytics changes in a separate PR? They don't seem directly related to adding a USPS confirmation page, and I think this requires more thought about what distinction we want to make here. IdV completion could immediately follow user registration for example.